nixos/modules/services/networking/i2pd.nix

```diff
@@ -698,8 +781,39 @@ in
       {
         User = "i2pd";
         WorkingDirectory = homeDir;
-        Restart = "on-abort";
         ExecStart = "${cfg.package}/bin/i2pd ${i2pdFlags}";
+        ## Auto restart
+        Restart = if cfg.autoRestart then "on-failure" else "no";
```

Comment:
A service should always be restarted unless we have a good reason not to

Reply:
One of i2pd use-cases is anonymity, and a failure may be a sign of a security breach, so I'm not sure if autorestart should be the default.

Reply:
A crash can also be a sign of to little memory, so not restarting may cause outages. Also I am not sure what nixos-rebuild will do with that on top of my mind.